dev-docs/bundle-rfc.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2019-03-12
+- RFC PR: [#408](https://github.com/uber/xviz/pull/408)
+- XVIZ Issue: [#250](https://github.com/uber/xviz/issues/250)
+
+# Summary
+
+We make the PNG and GIF of XVIZ. By making a `xviz/bundle` type that combines a `metadata` field
```

Comment:
just to be clear, the `xviz/bundle` is the envelope type correct.

Reply:
Yeah that is the envelope type I will make that more clear.  Along with updating to list the actual GLB implementation explicitly.